js/components/image.js

```diff
@@ -146,6 +156,17 @@ Fliplet.FormBuilder.field('image', {
     },
     processImage: function(file, addThumbnail) {
       var $vm = this;
+
+      var image = new Image();
+      image.onload = function () {
+        if (this.height > $vm.customHeight || this.width > $vm.customWidth) {
+          $vm.isImageValid = false;
+        } else {
+          $vm.isImageValid = true;
+        }
+      };
+      image.src = window.URL.createObjectURL(file);
```

Comment:
Can you wrap `file` into `Fliplet.Media.authenticate(file)`? I think it might be required to add the authorization token to the files if their URL is remote.